swift/README.md

```diff
@@ -1,36 +1,43 @@
-# AWS SDK for Swift code examples 
+# AWS SDK for Swift code examples
+## Overview
+The code examples in this topic show you how to use the AWS SDK for Swift with AWS.
 
-## Purpose
+The AWS SDK for Swift provides a Swift API for AWS infrastructure services. Using the SDK, you can build applications on top of Amazon S3, Amazon EC2, Amazon DynamoDB, and more.
 
-These examples demonstrate how to use the AWS SDK for Swift to access and use
-Amazon Web Services from your projects.
+## Types of code examples
+* **Single-service actions** - Code examples that show you how to call individual service functions.
 
-## Prerequisites
+* **Single-service scenarios** - Code examples that show you how to accomplish a specific task by calling multiple functions within the same service.
 
-You must have an AWS account, and have configured your default credentials and
-AWS Region as described in the [Getting started
-section](https://docs.aws.amazon.com/sdk-for-swift/latest/developer-guide/getting-started.html)
-of the _AWS SDK for Swift Developer Guide_.
+* **Cross-service examples** - Sample applications that work across multiple AWS services.
 
-## Running the code
+Single-service actions and scenarios are organized by AWS service in the
+[example_code](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/swift/example_code/)
+directory. A README in each folder lists and describes how to run the examples.
 
-See the individual readme files in each service directory for information about specific code examples for that service.
+Cross-service examples are located in the [cross_service](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/swift/cross_service/)
```

Comment:
You don't seem to have these yet so your link is 404. Here's what I did for Ruby in the meantime:
```
There are currently no cross-service examples for the AWS SDK for Ruby. To request a cross-service example, create an issue in the [AWS SDK Code Examples](https://github.com/awsdocs/aws-doc-sdk-examples/) repo.
```

Reply:
Good call. Changed.